### PR TITLE
Improve filter documentation

### PR DESCRIPTION
**Pull request by DaRaFF**

### CODE REVIEW

File: reference-docs/editor-configuration/menu-and-dashboards.md [hunk 1]
```diff
@@ -116,61 +116,19 @@ Note that the custom component can only use document metadata that has been expl
...
-
-- emptySearchQueries
-
-  This setting determines the empty search filter. This filter is taken into
-  account when there is no search query input present. Note that this filter is
-  treated mutually exclusive with the default filter that is used in case there is
-  a query present.
+Filters on a dashboard are highly customisable in Livingdocs.
 
+- One can define the order of filters in a dashboard (`displayFilters`)
```
Comment (mokimo) [inline reference-docs/editor-configuration/menu-and-dashboards.md]:
```suggestion
- One can define filters in a dashboard (`displayFilters`)
```

File: reference-docs/editor-configuration/menu-and-dashboards.md [hunk 2]
```diff
@@ -116,61 +116,19 @@ Note that the custom component can only use document metadata that has been expl
...
+- One can use core filter or custom filter registered over the editors `coreApi` in the `displayFilters`.
```
Comment (mokimo) [inline reference-docs/editor-configuration/menu-and-dashboards.md]:
```suggestion
- One can use core filters provided by Livingdocs or custom filters registered over the editors `coreApi` in the `displayFilters`.
```

File: reference-docs/editor-configuration/menu-and-dashboards.md [hunk 3]
```diff
@@ -237,37 +225,103 @@ The core allows you to use the following values for `displayFilters`:
...
-The next section shows you how you can add your own custom filters to the ones defined in the core.
 
-The configuration also allows you to set `defaultQueries`. Those are hidden search queries that are appended to each search query of the user. In the `type` you can use the same values as described in the `displayFilters` (including your own custom filters). The `value` describes the (fixed) value you want to set for this hidden query. Example:
+## Filter Query Format
+
+There are different places, where one can define a filter query
+- baseFilters (dashboard property)
+- defaultQueries (filter group property)
+- emptySearchQueries (filter group property)
+- custom filter (your own filter added to `displayFilters`)
```
Comment (mokimo) [inline reference-docs/editor-configuration/menu-and-dashboards.md]: Can we add a small description of each filter-type? That makes it easier to understand what they are used for

For example (maybe a bit rewritten and shorter):
```
- `baseFilters`
The configuration also allows you to set `defaultQueries`. Those are hidden search queries that are appended to each search query of the user.
```

File: reference-docs/editor-configuration/menu-and-dashboards.md [hunk 4]
```diff
@@ -237,37 +225,103 @@ The core allows you to use the following values for `displayFilters`:
...
 - `category`: uses the channel configuration for categories to offer a multi-select box to filter for categories (OR filter)
 
-The next section shows you how you can add your own custom filters to the ones defined in the core.
 
-The configuration also allows you to set `defaultQueries`. Those are hidden search queries that are appended to each search query of the user. In the `type` you can use the same values as described in the `displayFilters` (including your own custom filters). The `value` describes the (fixed) value you want to set for this hidden query. Example:
+## Filter Query Format
+
+There are different places, where one can define a filter query
+- baseFilters (dashboard property)
+- defaultQueries (filter group property)
```
Comment (mokimo) [inline reference-docs/editor-configuration/menu-and-dashboards.md]: filter group property, am unsure what that means, see the comment below

**Comment by mokimo** [comment]
One more suggestion:

https://github.com/livingdocsIO/livingdocs/pull/273/files#diff-0456965c20e82b283fd6c623036f125aR332

We could add another paragraph here, because that costed me a bit of time to realize why my filters aren't working:

```
Make sure that your metadata is correctly in elastic search (reference to ES-mapping docs)
```

**Comment by DaRaFF** [comment]
# hello1

## hello2

### hello3

#### hello4

**Pull request merged**